plugins/woocommerce/src/Admin/Features/PaymentGatewaySuggestions/DefaultPaymentGateways.php

```diff
@@ -7,20 +7,46 @@
 
 defined( 'ABSPATH' ) || exit;
 
-use Automattic\WooCommerce\Admin\Features\OnboardingTasks\Init as OnboardingTasks;
-
 /**
  * Default Payment Gateways
  */
 class DefaultPaymentGateways {
+	/**
+	 * Priority is used to determine which payment gateway to recommend first.
+	 * The lower the number, the higher the priority.
+	 *
+	 * @var array
+	 */
+	private static $recommendation_priority = array(
+		'woocommerce_payments'                            => 1,
+		'woocommerce_payments:with-in-person-payments'    => 1,
+		'woocommerce_payments:without-in-person-payments' => 1,
+		'stripe'                                          => 2,
+		'woo-mercado-pago-custom'                         => 3,
+		// PayPal Payments.
+		'ppcp-gateway'                                    => 4,
+		'mollie_wc_gateway_banktransfer'                  => 5,
+		'razorpay'                                        => 5,
+		'payfast'                                         => 5,
+		'payubiz'                                         => 6,
+		'square_credit_card'                              => 6,
+		'klarna_payments'                                 => 6,
+		// Klarna Checkout.
+		'kco'                                             => 6,
+		'eway'                                            => 6,
```

Comment:
For `AU`, `eway` should come after `square` but since they have the same priority, it's not shown correctly:

<img width="738" alt="image" src="https://user-images.githubusercontent.com/3747241/219350537-4e895f81-b7ae-4b9d-88ec-89081cf06208.png">

I think it'll be better if we have all the gateways have their own priority number, so that we can be deliberate on the priorities. I'm not sure if you have any opinion on having the same priority, let me know what you think!

Reply:
> For AU, eway should come after square but since they have the same priority, it's not shown correctly:

Thanks! Good catch. @ilyasfoo 

> I think it'll be better if we have all the gateways have their own priority number, so that we can be deliberate on the priorities. I'm not sure if you have any opinion on having the same priority, let me know what you think!

I use the same priority for them because we don't have a specific order for all the gateways. For example, I'm not sure if we should put `square_credit_card` before `payubiz` or after. Do you think Silu can help with this?

Reply:
Changed `eway` priority in cbd8ba92028f1af12446c70d024414507cc2cede.

Reply:
> I use the same priority for them because we don't have a specific order for all the gateways. 🤔 For example, I'm not sure if we should put square_credit_card before payubiz or after. Do you think Silu can help with this?

Yeah, it's not entirely clear, but I think for those without contractual obligation it doesn't matter. It should be fine for now, we can always do a follow-up if required.